bundles/framework/download-basket/Basket.js

```diff
@@ -164,7 +164,7 @@ Oskari.clazz.define(
             };
             var strUserDetails = JSON.stringify(userDetails);
 
-            var ajaxUrl = me._sandbox.getAjaxUrl();
+            var ajaxUrl = Oskari.urls.getRoute();
```

Comment:
You could add 'DownloadInfo' as param here and remove it from the concatenated url below.